app/react-native/src/server/index.js

```diff
@@ -24,13 +24,6 @@ export default class Server {
       }
     }
 
-    // see https://github.com/websockets/ws/issues/1256#issuecomment-364988689
-    socket.on('error', err => {
-      // Ignore network errors like `ECONNRESET`, `EPIPE`, etc.
-      if (err.errno) return;
-      throw err;
-    });
-
```

Comment:
Not needed anymore, see https://github.com/websockets/ws/issues/1256#issuecomment-370801041